misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/step_scan/computeStepScanPropsFromActivity.js

```diff
@@ -5,7 +5,7 @@ import {
   getStepByIdFromLine,
 } from '../../../../../reducers/wfProcesses';
 
-export const computeStepScanPropsFromActivity = ({ activity, lineId, stepId }) => {
+export const computeStepScanPropsFromActivity = ({ activity, lineId, stepId, isProcessedQtyStillOnScale }) => {
```

Comment:
what's the meaning / purpose of `isProcessedQtyStillOnScale` ?

what's `qtyAlreadyOnScale` and how that shall be used?

Reply:
if that's true, it means the already issued qty is still on the scale, so the process needs to take that into consideration before issuing, i.e. qtyComingFromScale - qtyAlreadyIssued = qtyToIssueNow